packages/link-modules/src/services/link-module-service.ts

```diff
@@ -188,6 +202,21 @@ export default class LinkModuleService<TLink> implements ILinkModule {
 
     const links = await this.linkService_.create(data, sharedContext)
 
+    await this.eventBusModuleService_?.emit<Record<string, unknown>>(
+      (data as { id: unknown }[]).map(({ id }) => ({
+        eventName: this.entityName_ + ".attached",
```

Comment:
nit: Use `CommonEvents`
```suggestion
        eventName: this.entityName_ + "." + CommonEvents.ATTACHED,
```